packages/messaging/index.js

```diff
@@ -17,12 +17,15 @@
  *
  * - Windows: {@link WindowsMessagingConfig}
  * - Webkit: {@link WebkitMessagingConfig}
+ * - Android: {@link AndroidMessagingConfig}
```

Comment:
(Random line since GitHub doesn't allow for inline comments on unchanged lines/files.)

Sounds good about trying to keep this change small as possible. But I think we should at least add comments to the inject/android.js code to say that it's deprecated and to point at the replacement.

Reply:
Agreed, thanks :)